README.md

```diff
@@ -13,6 +13,8 @@ pip install -U datasets
 pip install -e .
 ```
 
+> Note: If you wish to use [FlashAttention](https://github.com/Dao-AILab/flash-attention), then you need to install these requirements: `pip install -r flashattn_requirements`. [FlashAttention](https://github.com/Dao-AILab/flash-attention) requires the [CUDA Toolit](https://developer.nvidia.com/cuda-toolkit) to be pre-installed.
```

Comment:
I thought that `pip install flash-attn` does not require CUDA toolkit, because I have installed it in docker containers with no CUDA toolkit before. 

My experience is only that `pip install flash-atten --no-bulid-isolation` will run the compile process (and therefore require toolkit). The above command simply installs pre-compiled binaries so no toolkit should be required.